eip-0018.md

```diff
@@ -0,0 +1,387 @@
+# ErgoFund contracts
+
+* Author: code-for-uss, kushti
+* Status: Proposed
+* Created: 5-Nov-2021
+* License: CC0
+* Forking: not needed 
+
+## Motivation 
+
+Collecting funds in different contexts is needed to build common infrastructure and applications in the Ergo ecosystem. Therefore, this EIP is proposing contracts and standardized box formats for announcing crowdfunding campaigns and collecting funds.
+
+## Overall Design
+
+The design of the crowdfunding contracts and box templates below is centered by a seamless on-chain and off-chain code built on top of the Ergo Playgrounds, [ErgoFund scenarios via playground](https://scastie.scala-lang.org/TWlB0hncT5SdYcTax3Wj5A). 
+
+There is an old version of ErgoFund eip [here](#old-ergofund-contracts), prepared by Kushti.
+
+## Tokens
+
+The ErgoFund has the following types of tokens.
+
+| Token | Issued quantity | purpose | where stored |
+|---|---|---|---|
+|ControlBoxNFT | 1 | Identify control box | Any  |
+|SellNFT | 1 | Identify sell box | Sell box |
+|Registration tokens | 100 million | Identify each campaign box | Sell box <br /> Campaign boxes |
+|Campaign participant tokens | 100 million per campaign | Identify campaign and each pledge box| Capmaign box <br /> pledge boxes |
+
+## Boxes
+
+There are a total of 3 contracts, and each corresponds to a box-type + control box.
+
+| Box | Quantity | Tokens | Additional registers used | Purpose | Spending transactions |
+|---|---|---|---|---|---|
+|Control | 1 | ControlBoxNFT | R4: Price (Long), <br /> R5: address[script] (SigmaProp) | Control price and address to pay for campaign registration | <div align="center">-</div> | 
+|Sell| 1 | SellNFT <br /> Registration tokens | <div align="center">-</div> | Register a new campaign | registerCampaign, <br/> returnRegistrationToken |
```

Comment:
Why would you need to return the registration token since this box is already a bottleneck for starting crowdfunding? I guess you could burn the token instead or send it to the `false` script.

Reply:
I am opposed to the idea of having one sell box. There is no reason for that in this case (unlike cases like SigmaUSD bank box)
No reason to create a bottleneck when there is no gain.